src/huggingface_hub/inference/_generated/_async_client.py

```diff
@@ -734,6 +741,41 @@ async def chat_completion(
             print(chunk.choices[0].delta.content)
         ```
 
+        Example using Image + Text as input:
+        ```py
+        # Must be run in an async context
+        >>> from huggingface_hub import AsyncInferenceClient
+
+        # provide a remote URL
+        >>> image_url ="https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg"
+        # or a base64-encoded image
+        >>> image_path = "/path/to/image.jpeg"
+        >>> with open(image_path, "rb") as f:
+        ...     base64_image = base64.b64encode(f.read()).decode("utf-8")
+        >>> image_url = f"data:image/jpeg;base64,{base64_image}"
+
+        >>> client = AsyncInferenceClient("mistralai/Mistral-Nemo-Instruct-2407")
```

Comment:
should this be pixtral? nemo instruct is text-only

Reply:
Definitely an oversight from me. It seems that when the model is an LLM (not VLM), the data is still transferred to as text. In my case the URL contains "statue of liberty" so the LLM has been able to describe it (and trick me into thinking the image has been used).

Reply:
LLM: 1
Humans: 0